core/templates/dev/head/app.js

```diff
@@ -40,17 +40,18 @@ oppia.constant(
 oppia.constant('PLACEHOLDER_OUTCOME_DEST', '/');
 oppia.constant('INTERACTION_DISPLAY_MODE_INLINE', 'inline');
 oppia.constant('DEFAULT_RULE_NAME', 'Default');
```

Comment:
Optional: I also wonder if we need this, or if we can do something like rule.isDefault() instead? No need to worry about it for this PR if it's not trivial to replace.